index.html

```diff
@@ -1,9 +1,543 @@
+        
+                <nav class="wrapperright-link">
+                    <ul>
+                        <li><a href="#">About</a></li>
+                        <li><a href="#">Advertise</a></li>
+                        <li><a href="#">Jobs</a></li>
+                        <li><a href="#">Contact</a></li>
+                    </ul>
+                </nav>
+            </div>
+            <div class="headersecond-row">
+                <nav>
+                    <ul>
+                            
+                        <li><a href="#"><img src="images/favicon-96x96.png" alt="#"></a></li>
+                        <li><a href="#">latest</a></li>
+                        <li><a href="#">hard work</a></li>
+                        <li><a href="#">plugged</a></li>
+                        <li><a href="#">fundementals</a></li>
+                        <li><a href="#">readme</a></li>
+                        <li><a href="#">growth quarters</a></li>
+                        <li><a href="#">neural</a></li>
+                    </ul>
+                </nav>
+                <div>
+                    <p><i class="fas fa-search"></i></p>
+                </div>
+                <div class="three-bars">
+                    <i class="fas fa-bars"></i>
+                </div>
+            </div>
+
+            <div class="header-lastrow yellow-bar">
+                <span>We heard you like events and sweatpants so we made events you can join in your sweatpants ➔</span>
+            </div>
+            
+        </header>
+        <main class="wrapper">
+            <!-- <div class="presentation">
+                <article class="cover-image firstpresetation-bloc ">
+                    <h1>This hybrid AI system can <br/> understand causality in controlled<br/> environments</h1>
+                    
+                    <p>by <a href="#" class="link-white">Ben Dickson</a><i class="far fa-clock"></i> 6 hours ago <i class="fas fa-share-alt"></i> 0
+                    </p>
+                </article>
+                <div class="secondpresetation-bloc cover-trigger">
+                    <article>
+                        <h1>Shopify, Basecamp, and more will<br/>discuss the future of ecommerce<br/>at Sprint on June 11</h1>
+                    </article>
+                    <article>
+                        <div><a href="#">Tech</a></div>
+                        <h1>Can dark matter be found on<br/> earth? New research says ‘maybe’</h1>
+                        <ul>
+                            <li>
+                                by
+                                <a href="#" class="link-white">The Cosmic Companion</a>
+                            </li>
+                            <li><a href="#" class="link-white"><i class="far fa-clock"></i> 10 hours ago</a></li>
+                            <li>
+                                <i class="fas fa-share-alt link-white"></i> 0
+                            </li>
+                        </ul>
```

Comment:
We have to remove all the commented codes. So, remove all the comments from the HTML and CSS file.

Reply:
ok, I removed these comments